SmartDeviceLink/SDLMutableDataQueue.h

```diff
@@ -0,0 +1,35 @@
+//
+//  SDLMutableDataQueue.h
+//  
+//
+//  Created by David Switzer on 3/2/17.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@interface SDLMutableDataQueue : NSObject
+
+- (instancetype)init;
```

Comment:
Yes, we do not write the inherited initializers in the public interface. There may be some older interfaces that still have that, but they will be removed in the future.